js/try.js

```diff
@@ -94,6 +97,58 @@ const setCache = ({
   }
 }
 
+const doDownload = async (downloadButton) => {
+  try {
+    const {
+      identifier,
+    } = getCache();
+
+    let classes = [];
+    for (let i = 0; i <= 35; i++) { // say there's 35 weeks
```

Comment:
A full year is 44 weeks based on the university's timetable

Reply:
good point :)